ansible_runner/config/_base.py

```diff
@@ -314,6 +314,14 @@ def _update_volume_mount_paths(self, args_list, src_mount_path, dest_mount_path=
 
         self._ensure_path_safe_to_mount(src_mount_path)
 
+        def _add_trailing_slash_if_neeeded(some_path):
```

Comment:
Can we mode this method as level class method? IMO defining a method within a method makes code less readable.

Reply:
Sure thing!